Updated Readme - added note about maven compile target failure (#160)
confluentinc/parallel-consumer: src/docs/README.adoc

git bisect run ./bisect.sh
----

> Note::

Reviewer: did you make a GH issue for this to be fixed as well? If not please do so. And link to the GH issue from the note in the readme.

git bisect run ./bisect.sh
----

Note::

> mvn compile - Due to a bug in maven handling of test-jar dependencies - running `mvn compile` fails,

Reviewer: ```suggestion
`mvn compile` - Due to a bug in Maven's handling of test-jar dependencies - running `mvn compile` fails, Use `mvn test-compile` instead.
```

Author: updated

unless `mvn install` or `mvn deploy` was executed previously to create test-jar of parallel-consumer-core.

> See https://github.com/confluentinc/parallel-consumer/issues/162

Reviewer: ```suggestion
See https://github.com/confluentinc/parallel-consumer/issues/162[issue #162]
```

Author: updated.

> and https://stackoverflow.com/questions/4786881/why-is-test-jar-dependency-required-for-mvn-compile

Reviewer: ```suggestion
and this https://stackoverflow.com/questions/4786881/why-is-test-jar-dependency-required-for-mvn-compile[Stack Overflow question].
```

Author: updated.
